### PR TITLE
App 997: Pass an update robot part function to the call queue

### PR DESCRIPTION
**Pull request by RoxyFarhad**
This is required for this work: https://github.com/viamrobotics/app/pull/2175

Went through a lot of iterations for this:
1. using the signaling answer functionality but that only indicated when the connection was created + dropped
2. Creating a change stream for the operator table (complex set up)

This definitely made the most sense, and was the most concise change

### CODE REVIEW

File: rpc/wrtc_call_queue_mongodb.go [hunk 1]
```diff
@@ -87,6 +87,8 @@ type mongoDBWebRTCCallQueue struct {
 	csStateUpdates              chan changeStreamStateUpdate
 	csCtxCancel                 func()
 
+	// function to update access times on robot parts based on this call queue
+	updateRobotPartConnectivity *func(hostnames []string)
```
Comment (michaellee1019) [inline rpc/wrtc_call_queue_mongodb.go:91]: Lets not name it app specific. 
```suggestion
	activeAnswerersfunc *func(hostnames []string)
```

**Review by michaellee1019 — APPROVED** [review]
Just a naming concern and the rest LGTM!

File: rpc/wrtc_call_queue_mongodb_test.go [hunk 2]
```diff
@@ -14,14 +14,33 @@ import (
 	"go.viam.com/utils/testutils"
 )
 
+var activeAnswerersTrigger chan int
+
+func init() {
+	activeAnswerersTrigger = make(chan int, 10)
+}
+
+func activeAnswerersStub(hostnames []string) {
```
Comment (RoxyFarhad) [inline rpc/wrtc_call_queue_mongodb_test.go:23]: @michaellee1019 this was my way to test that passing in a function would be called when running the call_queue. Not sure if a channel was the best way of going here but essentially I just wanted something that would be able to be pinged inside the operator goroutine.

File: rpc/wrtc_call_queue_mongodb_test.go [hunk 3]
```diff
@@ -153,4 +155,31 @@ func TestMongoDBWebRTCCallQueueMulti(t *testing.T) {
 		t.Logf("received offer %d=%s", (maxHostAnswerersSize*2)+1, exchange.UUID())
 		exchanges[0] = exchange
 	})
+
+	t.Run("ActiveAnswerer", func(t *testing.T) {
```
Comment (RoxyFarhad) [inline rpc/wrtc_call_queue_mongodb_test.go:159]: @michaellee1019 this is the updated test that creates a go routine with a single answerer

Comment (michaellee1019) [reply rpc/wrtc_call_queue_mongodb_test.go:159]: Nice! good job figuring this out. I like the simple length test.

**Review by michaellee1019 — APPROVED** [review]

**Pull request merged**